modules/picarto.js

```diff
@@ -1,4 +1,4 @@
-﻿'use strict';
+'use strict';
```

Comment:
Well, fuck me then. Figured that it must have been some kind of invisible utf-8 control character thingie. It's cool, thanks for removing that.